ocfweb/docs/templates/docs/officers.html

```diff
@@ -80,6 +80,26 @@ <h4>
           the OCF lab and other services. They are elected by the Board of
           Directors at the end of each semester.
         </p>
+      {% if current_term.heads|length > 0 %}
```

Comment:
Is it possible to just say `{% if current_term.heads %}`?

Reply:
Not totally sure, but this is following the pattern of lines 36 and 65. I'll try to test it.